pkg/machine/ignition.go

```diff
@@ -495,24 +499,13 @@ Delegate=memory pids cpu io
 		if _, err := os.Stat(sslCertFile); err == nil {
 			certFiles = getCerts(sslCertFile, false)
 			files = append(files, certFiles...)
+		}
+	}
 
-			if len(certFiles) > 0 {
```

Comment:
removed because SSL_CERT_FILE will be written to `/etc/profile.d/default-env.sh` by `envset.service`